### ISSUE TITLE
ScrollablePane with DetailsList does not show (virtualized) items

### ISSUE THREAD

**Issue by obiwan007** [OfficeDev/office-ui-fabric-react#4204]
### Bug Report

- __Package version(s)__: 5.60.0
- __Browser and OS versions__: Chrome, Edge (latest)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Blocking) 

Products/sites affected: (if applicable)

Demo App, ScrollablePane Demo, <ScrollablePaneDetailsListExample>

#### Describe the issue:

Scrolling down the items list, does not show all items. Instead the list will stay blank after a few top items.
As the DetailsList is virtualizing the dom items it seem that inside the ScrollableView container, the item recycle is not triggered anymore.

#### Actual behavior:

Not all items are show in the DetailsList

#### Expected behavior:

All items are show in the DetailsList

### If applicable, please provide a codepen repro:

See npm start 
demo component page.

**Comment by Jahnp** [comment]
Thanks for reporting this @obiwan007 . Just so we understand since you marked this blocking, are you just reporting an issue with the demo, or are you looking to implement a similar scenario?

**Comment by obiwan007** [comment]
I mentioned the demo app just as an example as you could easily reproduce it there.
For us it is a critical feature as currently the list is not showing all items anymore..

**Comment by aappddeevv** [comment]
You can turn off virtualization as a workaround `onShouldVirtualize = { () => false}` in case of emergencies. I ran into this issue in earlier versions when freezing the size of the underlying DetailsList in order to force the header to not move....a technique that did not work and made obsolete by the ScrollablePane/Sticky header.

**Comment by antonafMSFT** [comment]
I've run into a similar issue today.

@aappddeevv I believe the fix you reference did fix the issue, but it was broken again by #4055.

The underlying issue is that after #4055 the content of the `ScrollablePane` is put into a `div` that has `data-is-scrollable={ true }`, but is itself NOT scrollable - instead, its parent has the scrollbars.
This causes the `List` (and `DetailsList` by extension) to never receive a `scroll` event from the `div` that has `data-is-scrollable={ true }`, and thus it never fills in the virtual items.

The fix could be simply moving the `data-is-scrollable` attribute to the outermost `div` of the `ScrollablePane`, but this might end up breaking the very thing that #4055 was trying to fix.

**Comment by aappddeevv** [comment]
I understand your comment. I was running on 5.62 and I do get a full render of content. However, I've run into another issue with the sticky header model and vertical scrolling. #4234

**Comment by obiwan007** [comment]
![microsoft edge 11 04 2018 09_37_01](https://user-images.githubusercontent.com/12936285/38602763-ff01d64a-3d6b-11e8-8b2f-767b0d70d409.png)
Unfortunately this bug is still not fixed. Very strange that nobody else notices that the basic concept of the virtualized list is not working anymore.
You could check it in the basic components page under ScrollablePane. You will find that the Items List there is only showing a few items and is not triggering a refeed of new items. If you force redraw (resize browser) - all is fine...

For now I just switched off the virtualized feature but that can't be the final solution.

**Comment by Jahnp** [comment]
@leddie24  is working on a refactor of ScrollablePane, which may address this issue. @leddie24 can you check if this will be fixed by #4491?

**Comment by leddie24** [comment]
Yup, it will be fixed.  Basically the scroll event handler isn't firing because the wrong container has data-is-scrollable=true.  This will be fixed with that PR

**Comment by jordandrako** [comment]
Did #4602 fix this?

**Comment by russgove** [comment]
Hi, 
I'm having a similar issue. I am using a DetailsList  in an SPFX  ListView CmmandSet. The DetailsList does not render until I resize the window.  I tried adding 
  onShouldVirtualize = { () => false}

but it did not help.
I am using office-ui-fabric-react@5.41.2

any ideas?

**Comment by nealYangVic** [comment]
onShouldVirtualize = { () => false} do fix the problem, but the giant data will be very slow....

**Comment by KevinTCoughlin** [comment]
> onShouldVirtualize = { () => false} do fix the problem, but the giant data will be very slow....

Can you intercept `resize` events on your container and call `forceUpdate` on the `List` component ref as a workaround? Similar to how `List` monitors `window` `resize`, see https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/List/List.tsx#L297.

**Comment by nealYangVic** [comment]
> > onShouldVirtualize = { () => false} do fix the problem, but the giant data will be very slow....
> 
> Can you intercept `resize` events on your container and call `forceUpdate` on the `List` component ref as a workaround? Similar to how `List` monitors `window` `resize`, see https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/List/List.tsx#L297.

Thanks, but I think after rendering, scrolling on the list will still be very slow

**Comment by nealYangVic** [comment]
Actually my issue is scroll issue related with virtualized, after below, my problem fixed

<ScrollablePane>
                <DetailsList onRenderDetailsHeader={
                        // tslint:disable-next-line:jsx-no-lambda
                        (detailsHeaderProps, defaultRender) => (
                          <Sticky>
                            {defaultRender(detailsHeaderProps)}
                          </Sticky>
                        ) }
/>
</ScrollablePane>
